be/src/exec/base_scanner.cpp

```diff
@@ -178,6 +178,11 @@ Status BaseScanner::init_expr_ctxes() {
                 if (_src_slot_it == std::end(src_slot_desc_map)) {
                     return Status::InternalError("No src slot {} in src slot descs", it1->second);
                 }
+                auto src_slot_index = std::find(_src_slot_descs.cbegin(), _src_slot_descs.cend(),
```

Comment:
Maybe we can maintain a map represent <slot_id -> slot_index>, to prevent find desc in vector everytime.

Reply:
done. thanks